packages/client/src/components/Location/LocationSummary.test.tsx

```diff
@@ -171,90 +169,4 @@ describe("LocationSummary", () => {
       ).not.toBeInTheDocument();
     });
   });
-
-  /**
-   *
-   * OLO FLOW START HERE:
-   *
-   */
-
```

Comment:
All these tests were already covered by my OloLocationInput and OloLocationResult tests